core/audits/audit.js

```diff
@@ -214,17 +214,23 @@ class Audit {
   /**
```

Comment:
Meta comment (applies everywhere): sorry this will be annoying but the vast majority of our comments fit this style:

```
// Capitalized the first letter and ends in a period.
```

Can you match?

Reply:
Yeah, not an issue. I'll scan and update.

Reply:
I did scan; not sure if I missed any, but it looks like I've matched all comments to adjacent lines' style and defaulted to the style you mentioned already. Did you find a specific instance?